gkShell/gk.sh

```diff
@@ -0,0 +1,507 @@
+}
+command-images() {
+  echo "docker images ${IMAGE_NAME}"
+  ${DOCKER_CMD} images ${IMAGE_NAME}
+}
+command-help-history() {
+ echo " show commands history for geokrety docker images"
+}
+image-history() {
+  echo "docker history ${IMAGE_FULLNAME}"
+  ${DOCKER_CMD} history ${IMAGE_FULLNAME}
+}
+# command-run-port-only() {
+#   echo "docker run -tid --name ${CONTAINER_NAME} -p 80:80 ${IMAGE_FULLNAME}"
+#   docker run -tid --name ${CONTAINER_NAME} -p 80:80 ${IMAGE_FULLNAME}
+# }
+# command-run-volume() {
+#   echo "docker run -tid --name ${CONTAINER_NAME} -p 80:80 --mount source=${VOLUME_NAME},target=${MOUNT_TARGET} ${IMAGE_FULLNAME}"
+#   docker run -tid --name ${CONTAINER_NAME} -p 80:80 --mount source=${VOLUME_NAME},target=${MOUNT_TARGET} ${IMAGE_FULLNAME}
+# }
+command-help-run() {
+ echo " run geokrety containers from: geokrety docker image, and mariadb image"
+}
+command-run() {
+  if isWindowsHost; then
+    command-run-windows
+  else
+    command-run-linux
+  fi
+}
+command-run-windows() {
+  computedParams
+  if ! hasWebsiteMapping || ! hasConfigMapping; then
+    echo " X please run $0 install";
+    exit 1
+  fi
+  DOCKER_IP=$(docker-machine ip ${DOCKER_DEFAULT_MACHINE})
+  echo "docker run ${DB_CONTAINER_NAME}"
+  ${DOCKER_CMD} run -tid --name ${DB_CONTAINER_NAME} -p 3306:3306 \
+      -v /${SHARED_FOLDER_GKCONFIG}/mariadb:/docker-entrypoint-initdb.d \
+      -e "MYSQL_ROOT_PASSWORD=password" \
+      -e "TIME_ZONE=UTC" \
+      mariadb:10.3
+
+  DB_IP=$(${DOCKER_CMD} inspect --format '{{ .NetworkSettings.Networks.bridge.IPAddress }}' ${DB_CONTAINER_NAME})
+  echo "alias db:${DB_IP}"
+
+  echo "docker run ${ADM_CONTAINER_NAME}"
+  ${DOCKER_CMD} run -tid --name ${ADM_CONTAINER_NAME} -p 8880:8080 \
+      --add-host="db:${DB_IP}" \
+      adminer:4.6.2
+
+  echo "docker run ${CONTAINER_NAME}"
+  VOLUME_OPTS="-v /${SHARED_FOLDER_GKWEBSITE}:${CONTAINER_MOUNT_TARGET}"
+  VOLUME_OPTS="${VOLUME_OPTS} -v /${SHARED_FOLDER_GKCONFIG}:${CONTAINER_MOUNT_CONFIG}"
+  if [ -d "${SCRIPTS_DIR}" ]; then
+    VOLUME_OPTS="${VOLUME_OPTS} -v /${SHARED_FOLDER_GKSCRIPTS}:${CONTAINER_MOUNT_SCRIPTS}"
+  fi
+
+  ${DOCKER_CMD} run -tid --name ${CONTAINER_NAME} -p 80:80 ${VOLUME_OPTS} \
+      -e "website_config_directory=${CONTAINER_MOUNT_CONFIG}" \
+      --add-host="db:${DB_IP}" \
+      ${IMAGE_FULLNAME}
+
+  echo
+  echo " - 'C:\Windows\System32\drivers\etc\hosts' aliases to verify:"
+  echo "     ${DOCKER_IP}  default-machine"
+  echo
+  echo "lets go !"
+  echo "     Adminer    - http://default-machine:8880/?server=default-machine&username=root&db=geokrety-db"
+  echo "     Geokrety   - http://default-machine/"
+  echo "NB/ you could have to wait a minute while database init."
+}
+command-run-linux() {
+  computedParams
+  echo "docker run ${DB_CONTAINER_NAME}"
+  ${DOCKER_CMD} run -tid --name ${DB_CONTAINER_NAME} -p 3306:3306 \
+      -v ${HOST_GK_CONFIG}/mariadb:/docker-entrypoint-initdb.d \
+      -e "MYSQL_ROOT_PASSWORD=password" \
```

Comment:
Hard-coded password

Reply:
Yep, I will continue to improve docker-compose file and fwd environment (or use env file) following [this doc](https://docs.docker.com/compose/environment-variables/)

Reply:
`.env` file seems not working, so I enforce user to have (exported) `MYSQL_ROOT_PASSWORD` as requirement. I will keep `konfig-mysql.php` which rely on environment too to avoid having hard coded password (push to come)

Reply:
done using template and env